cmd/loki/loki-local-config.yaml

```diff
@@ -18,9 +18,6 @@ common:
     kvstore:
       store: inmemory
 
-ingester_rf1:
```

Comment:
this appears to have been removed from the code, but was not removed from the config? I got an error on `main` when trying to use this config.